### PR TITLE
Account Creation by Activation Code

### PR DESCRIPTION
**Pull request by dhakim87**
Lets user enter an Activation Code instead of a Kit ID to login.  

Probably shouldn't merge this until there is actually a way to distribute activation codes to users??

### CODE REVIEW

File: microsetta_interface/implementation.py [hunk 1]
```diff
@@ -1203,6 +1205,20 @@ def get_ajax_list_kit_samples(kit_name):
     return flask.jsonify(result), code
 
 
+def get_ajax_check_activation_code(code, email):
+    response = requests.get(
+        ApiRequest.API_URL + '/can_activate',
+        auth=BearerAuth(session[TOKEN_KEY_NAME]),
+        verify=ApiRequest.CAfile,
+        params=ApiRequest.build_params({"email": email, "code": code}))
```
Comment (wasade) [inline microsetta_interface/implementation.py:1213]: I don't think it matters if the email address is sent, just that it's a valid code?

Comment (wasade) [reply microsetta_interface/implementation.py:1213]: ...if the email is sent, then we would need to prompt the user to make sure they use the same email for which they received an activation code for

**Review by wasade — COMMENTED** [review]
Question on emails, otherwise this looks good!

**Comment by dhakim87** [comment]
This PR is now updated to allow sending activation emails one at a time through the admin toolbar as activation codes are generated.

**Pull request merged**